package/dart-sass.sh

```diff
@@ -8,12 +8,15 @@
 # package`.
 
 function follow_links() {
-  file="$1"
+  local file="$1"
+
+  cd $(dirname "$file")
   while [ -h "$file" ]; do
     # On Mac OS, readlink -f doesn't work.
     file="$(readlink "$file")"
+    cd $(dirname "$file")
   done
-  echo "$file"
+  echo "$(pwd -P)/$(basename "$file")"
```

Comment:
This script isn't deployed on Mac. /usr/local/bin/dart is directly a symlink to the binary.

Concerning this fix, Homebrew uses double relative links redirection. Without the fix, the issue is following:
```
$ dart-sass 
/usr/local/bin/dart-sass: line 21: <current pwd>/../Cellar/dart-sass/1.0.0-alpha.8/bin/src/dart: No such file or directory
/usr/local/bin/dart-sass: line 21: exec: <current pwd>/../Cellar/dart-sass/1.0.0-alpha.8/bin/src/dart: cannot execute: No such file or directory
```